<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1068</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
* {	border-color: lightgrey;
	background-color: black;
	color:white;
}
QFrame[frameShape=&quot;4&quot;][frameShadow=&quot;48&quot;],
QFrame[frameShape=&quot;5&quot;][frameShadow=&quot;48&quot;] {
	border: none;
    background: lightgrey;
}
QFrame[frameShape=&quot;4&quot;][frameShadow=&quot;48&quot;] {
	width: 1px;
}
QFrame[frameShape=&quot;5&quot;][frameShadow=&quot;48&quot;] {
	height: 1px;
}
QTextEdit {font:&quot;Consolas&quot;}
QToolTip {color: black;}
QPushButton {color:white; background-color: black;}
QHeaderView::section {
	border:1px, black;
	color:white; 
	background-color: darkgrey;
}
PinView {
	selection-color: darkblue;
	selection-background-color: deepskyblue;
}
QDialog{}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="leftFrame">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="signalFrame">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QFrame" name="inputPinFrame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="inputPinLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Input Pin</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="PinView" name="inputPinTable">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="outputPinFrame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="outputPinLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Output Pin</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="PinView" name="outputPinTable">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="leftHorLine">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="hdlFrame">
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="hdlCodeLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>HDL Code</string>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="hdlEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">color:white;
font: Consolus</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="verLine">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="rightFrame">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="simulationFrame">
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QFrame" name="miscFrame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="simLabel">
               <property name="text">
                <string>Simulation Code</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="simulationEdit">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="sliderFrame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QSlider" name="speedSlider">
               <property name="maximum">
                <number>4</number>
               </property>
               <property name="value">
                <number>2</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="speedLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>2</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="rightHorLine">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="animationFrame">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>200</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QFrame" name="internalPinFrame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="internalPinLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Internal Pin</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="PinView" name="internalPinTable">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="nameFrame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="nameLabel">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Chip Name:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPlainTextEdit" name="nameEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1068</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="openHdlAction"/>
    <addaction name="openSimAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编译</string>
    </property>
    <addaction name="reloadAction"/>
    <addaction name="singleStepAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="openHdlAction"/>
   <addaction name="openSimAction"/>
   <addaction name="singleStepAction"/>
   <addaction name="saveHdlAction"/>
   <addaction name="reloadAction"/>
  </widget>
  <action name="newHdlAction">
   <property name="text">
    <string>新建HDL文件</string>
   </property>
  </action>
  <action name="singleStepAction">
   <property name="text">
    <string>单步调试</string>
   </property>
   <property name="toolTip">
    <string>进行单步调试</string>
   </property>
  </action>
  <action name="openHdlAction">
   <property name="text">
    <string>打开HDL文件</string>
   </property>
   <property name="toolTip">
    <string>打开HDL文件</string>
   </property>
  </action>
  <action name="saveHdlAction">
   <property name="text">
    <string>保存HDL文件</string>
   </property>
   <property name="toolTip">
    <string>保存HDL文件</string>
   </property>
  </action>
  <action name="reloadAction">
   <property name="text">
    <string>刷新页面</string>
   </property>
   <property name="toolTip">
    <string>刷新页面</string>
   </property>
  </action>
  <action name="newSimAction">
   <property name="text">
    <string>新建仿真文件</string>
   </property>
   <property name="toolTip">
    <string>新建仿真文件</string>
   </property>
  </action>
  <action name="openSimAction">
   <property name="text">
    <string>打开仿真文件</string>
   </property>
   <property name="toolTip">
    <string>打开仿真文件</string>
   </property>
  </action>
  <action name="execSimAction">
   <property name="text">
    <string>运行仿真文件</string>
   </property>
   <property name="toolTip">
    <string>运行仿真文件</string>
   </property>
  </action>
  <action name="pauseAction">
   <property name="text">
    <string>暂停运行</string>
   </property>
   <property name="toolTip">
    <string>暂停运行</string>
   </property>
  </action>
  <action name="resetSimAction">
   <property name="text">
    <string>重置运行</string>
   </property>
   <property name="toolTip">
    <string>重置运行</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PinView</class>
   <extends>QTableView</extends>
   <header location="global">pinview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
